path_config.go

```diff
@@ -272,7 +273,8 @@ func (b *jwtAuthBackend) pathConfigWrite(ctx context.Context, req *logical.Reque
 	case config.JWKSURL != "":
 		keyset, err := jwt.NewJSONWebKeySet(ctx, config.JWKSURL, config.JWKSCAPEM)
 		if err != nil {
-			return logical.ErrorResponse(errwrap.Wrapf("error checking jwks_ca_pem: {{err}}", err).Error()), nil
+			b.Logger().Error("error checking jwks_ca_pem", "error", err)
```

Comment:
I realize it was like this before, but I think this log and error should be like `error checking jwks_url`. I guess it could be a problem with the CA too.. So maybe `error checking jwks_url or jwks_ca_pem`? 🤔

Reply:
It looks like NewJSONWebKeySet() returns an error if jwks url is empty or if there is an error with the CA. On line 273 we check that the jwks url is not empty so I think this log is correct.